### DISCUSSION TITLE
Snap to face and edges

### DISCUSSION THREAD

**Discussion by JulienHeijmans** [JulienHeijmans/quicksnap#27] [Potential Upcoming features]
I am myself considering this:
Allowing the user to simply snap to edge and faces, not just face centers and edge midpoints.
Not sure how useful this would be, how many people would actually use this.

Don't hesitate to reply here if you have an opinion.

**Comment by etma101** [comment]
thats a great idea! it would make process more intuitional. especially when moving an object along an axis then it meets and edge or face. common sense is there should be a snap point where move axis and the edge or face has crossed.

**Comment by adhhrg** [comment]
this is great, especially when you have large low poly object, this will be so good
